osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -181,6 +182,21 @@ def update_order_status(self, extraction_order):
         else:
             return False
 
+    def estimated_file_size(self, north=0, west=0, south=0, east=0):
```

Comment:
Is it possible to get named arguments without default arguments? I don't like positional arguments for coordinates because every one uses an other ordering, so it would be much error-prone.